### PR TITLE
ADEN-5157 Possibility to enable MOAT video tracking without sampling

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/lookup/prebid/priceParsingHelper.js [hunk 1]
```diff
@@ -3,8 +3,9 @@ define('ext.wikia.adEngine.lookup.prebid.priceParsingHelper', [
 	'ext.wikia.adEngine.utils.sampler',
 	'wikia.geo',
 	'wikia.instantGlobals',
-	'wikia.log'
-], function (sampler, geo, instantGlobals, log) {
+	'wikia.log',
+	'wikia.window'
+], function (sampler, geo, instantGlobals, log, win) {
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/lookup/prebid/priceParsingHelper.js:8]: MAJOR Function has 5 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 3 major

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [veles.js#L11](https://github.com/Wikia/app/blob/b3f43dfba0bb5ffe61adc88205285ebb394341d8/extensions/wikia/AdEngine/js/lookup/prebid/adapters/veles.js#L11): Function has 8 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![MAJOR][MAJOR] [porvata.js#L14](https://github.com/Wikia/app/blob/b3f43dfba0bb5ffe61adc88205285ebb394341d8/extensions/wikia/AdEngine/js/template/porvata.js#L14): Function has 11 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 3 major

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [veles.js#L11](https://github.com/Wikia/app/blob/4d827d2c0eeaa985ef8e0d234ac5bc6ddb1817e5/extensions/wikia/AdEngine/js/lookup/prebid/adapters/veles.js#L11): Function has 8 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![MAJOR][MAJOR] [porvata.js#L15](https://github.com/Wikia/app/blob/4d827d2c0eeaa985ef8e0d234ac5bc6ddb1817e5/extensions/wikia/AdEngine/js/template/porvata.js#L15): Function has 12 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 3 major

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [veles.js#L11](https://github.com/Wikia/app/blob/9e01884e7825b31578c7087c2182f900e9ef2b92/extensions/wikia/AdEngine/js/lookup/prebid/adapters/veles.js#L11): Function has 8 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![MAJOR][MAJOR] [porvata.js#L15](https://github.com/Wikia/app/blob/9e01884e7825b31578c7087c2182f900e9ef2b92/extensions/wikia/AdEngine/js/template/porvata.js#L15): Function has 12 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/AdEngine/js/video/videoSettings.js [hunk 2]
```diff
@@ -30,6 +34,25 @@ define('ext.wikia.adEngine.video.videoSettings', [
 			return Boolean(defaultStateAutoPlay || resolvedStateAutoPlay);
 		}
 
+		function isMoatTrackingEnabled(params) {
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/video/videoSettings.js]: We have a public method with the same name in this module. Is it possible to change name one of those?

**Review by kvas-damian — APPROVED** [review]
One comment. Except that it LGTM

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 3 major

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [veles.js#L11](https://github.com/Wikia/app/blob/24b9b48eb8709a340ae1002f7af5258e58dadd9f/extensions/wikia/AdEngine/js/lookup/prebid/adapters/veles.js#L11): Function has 8 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![MAJOR][MAJOR] [porvata.js#L15](https://github.com/Wikia/app/blob/24b9b48eb8709a340ae1002f7af5258e58dadd9f/extensions/wikia/AdEngine/js/template/porvata.js#L15): Function has 12 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**